profile.html

```diff
@@ -26,55 +32,74 @@
                 </div>
 
                 <div class="menuElements">
+                  <ul>
                     <li class="communityPage_title">Community</li>
-                    <li><a id="login">Logout</a></li>
-                    <li><a id="signUp">Profile</a></li>
-                    <li><a href="contact.asp">Settings</a></li>
-                    <li><a id="aboutPage">About</a></li>
+                    <li><a id="login" href="#">Logout</a></li>
+                    <li><a id="signUp" href="#" >Profile</a></li>
+                    <li><a href="settings.html" >Settings</a></li>
+                    <li><a id="aboutPage" href="#" >About</a></li>
                     <!-- have a search bar here  -->
+                  </ul>
                 </div>
```

Comment:
Like Alan said, we should not copy paste the nav bar on each page. We will only use ajax calls for the navbar. make an ajax call from the about page so that the contents of the about page change are replaced by the contents of the profile page. let me know in case you need help figuring out the ajax calls.